okd/install-packages.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#Script to install all packages to run the Keylime agent
+
+#To clear cached RPM repository metadata and perform a system upgrade
+rpm-ostree cleanup -m
+rpm-ostree upgrade
+
+# Updating base packages
+# `rpm-ostree install ...` adds packages that are not a part of the original OSTree as "LayeredPackages"
+# `rpm-ostree override replace ...` is used to update the base layer packages to a newer version using locally downloaded RPMs
+# Few of the packages that we need to install have requirements of newer verisons of base packages, so we need to manually download them.
+# Example: dbus-devel requires dbus-libs-1.12.20-1 instead of dbus-libs-1.12.18-1 which is a base package on FCOS
```

Comment:
I'm still not entirely clear on why we need to curl these packages individually by hand. I know we need newer versions than the base installs, but why can't rpm-ostree update them itself? This comment needs to address that.

Reply:
How about something like this:

We want to install several rpm packages, but they require newer versions of already installed packages. rpm-ostree isn't like a normal package manager and can't update already installed packages. So we need to manually download them and then `rpm-ostree override replace` them before we can `rpm-ostree install` the packages that need them.
Example: dbus-devel requires dbus-libs-1.12.20-1 instead of dbus-libs-1.12.18-1 which is a base package on FCOS